src/Krank/Checkers/IssueTracker.hs

```diff
@@ -138,7 +139,13 @@ headersFor issue = do
 httpExcHandler :: Localized GitIssue
                -> Req.HttpException
                -> ReaderT KrankConfig IO Value
-httpExcHandler issue _ = pure . AesonT.object $ [("error", AesonT.String . pack . show $ issueUrl . unLocalized $ issue)]
+httpExcHandler issue exc = pure . AesonT.object $ [("error", AesonT.String . pack $
```

Comment:
I found a bit surprising to do the error detection here and return an `Aeson` object. That's orthogonal to this issue, but in the future we may want to refactor that.

Reply:
agreed. I realized that as I was writing the new code